tests/test_end_to_end_pto_detector_non_pto_message.py

```diff
@@ -0,0 +1,169 @@
+      response = client.invoke(FunctionName=lambda_name,InvocationType='Event',Payload = pyloadobj,LogType='None')
+      return response
+
+  def get_query_id(self):
+    """
+    Method returns query id
+    """
+    query_id = None
+    while query_id == None:
+      client = boto3.client('logs')
+      start_query_response = client.start_query(logGroupName=cloudwatch_conf.pto_log_group,\
+        startTime=int((datetime.today() - timedelta(minutes=6)).timestamp()),\
+            endTime=int(datetime.now().timestamp()),queryString=cloudwatch_conf.query_pto_detector)
+      query_id = start_query_response['queryId']
+
+    return query_id
+
+  def get_message_from_pto_detector_logs(self,query_id):
+    """
+    Method to get message from pto_detector lambda insights
+    """
+    response = None
+    message = None
+    while response == None:
+      generic_utils.wait(1)
+      client = boto3.client('logs')
+      response = client.get_query_results(queryId=query_id)
+      response_dict = tests.helpers.cloudwatch_helper.get_data_structure(response)
+      if "results_0_1_value" in response_dict.keys():
+        message = response_dict['results_0_1_value']
+
+    return message
+
+  def get_logstream_from_ptr_value(self,query_id):
+    """
+    Method to fetch logstream ptr value
+    """
+    ptr_value = None
+    response = None
+    logstream = None
+    while response == None:
+      generic_utils.wait(1)
+      client = boto3.client('logs')
+      response = client.get_query_results(queryId=query_id)
+      response_dict = tests.helpers.cloudwatch_helper.get_data_structure(response)
+      logger.info(response_dict)
+      ptr_value = response_dict['results_0_2_value']
+      if ptr_value:
+        response = client.get_log_record(logRecordPointer=ptr_value)
+        response_dict = tests.helpers.cloudwatch_helper.get_data_structure(response)
+        logstream = response_dict['logRecord_@logStream']
+
+    return logstream
+
+  def get_query_id_from_flag_message(self):
+    """
+    Method to get query id from flag message
+    """
+    client = boto3.client('logs')
+    start_query_response = client.start_query(logGroupName=cloudwatch_conf.pto_log_group,\
+        startTime=int((datetime.today() - timedelta(minutes=7)).timestamp()),\
+            endTime=int(datetime.now().timestamp()),queryString=cloudwatch_conf.query_pto_detector_flag)
+    query_id = start_query_response['queryId']
+
+    return query_id
+
+  def get_logstream_from_flag_value(self,query_id):
+    """
+    Method to fetch logstream from flag value
+    """
+    response = None
+    logstream = None
+    while response == None:
+      generic_utils.wait(1)
+      client = boto3.client('logs')
+      response = client.get_query_results(queryId=query_id)
+      response_dict = tests.helpers.cloudwatch_helper.get_data_structure(response)
+      logstream = response_dict['results_0_2_value']
+
+    return logstream
+
+class ptodetectornonptotest(unittest.TestCase):
```

Comment:
I have added your comments in the below Trello ticket as I realized that these changes will be applicable to another test as well.
-https://trello.com/c/d46DdBcu/11-restructure-helper-functions-in-the-helper-folder-in-the-qxf2-lambda-tests-folder

Let me know in case you have any other comments.